code/modules/mob/language.dm

```diff
@@ -486,6 +486,14 @@
 	flags = RESTRICTED | HIVEMIND | NOBABEL
 	follow = TRUE
 
+/datum/language/terrorspider/broadcast(mob/living/speaker, message)
```

Comment:
You removed the third argument, but.. shouldn't it have it anyway? Since the parent proc has three arguments it's generally good practice to carry them over even if they won't be used.
```suggestion
/datum/language/terrorspider/broadcast(mob/living/speaker, message, speaker_mask)
```